src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -5869,6 +5869,14 @@ Copyright (C) Microsoft Corporation. All rights reserved.
   <!-- This import is temporary and will be removed once it is moved into the silverlight targets -->
   <Import Project="$(MSBuildToolsPath)\Microsoft.Data.Entity.targets" Condition="'$(TargetFrameworkIdentifier)' == 'Silverlight' and Exists('$(MSBuildToolsPath)\Microsoft.Data.Entity.targets')"/>
 
+  <!-- Import NuGet.targets (required for GetReferenceNearestTargetFrameworkTask and used for Restore functionality) -->
+  <PropertyGroup>
+    <NuGetRestoreTargets Condition="'$(NuGetRestoreTargets)'=='' and '$([MSBuild]::IsRunningFromVisualStudio())'=='true'">$(MSBuildToolsPath32)\..\..\..\Common7\IDE\CommonExtensions\Microsoft\NuGet\NuGet.targets</NuGetRestoreTargets>
+    <NuGetRestoreTargets Condition="'$(NuGetRestoreTargets)'==''">$(MSBuildToolsPath)\NuGet.targets</NuGetRestoreTargets>
```

Comment:
Is the location being changed from `$(MSBuildExtensionsPath)\NuGet.targets` ? (https://github.com/Microsoft/msbuild/blob/master//targets/bootstrapDependencies/Core/15.0/Microsoft.Common.targets/ImportAfter/Microsoft.NuGet.ImportAfter.targets#L15)

Reply:
Good catch, thanks! This shouldn't be needed at all anymore. This file was just because that was the way we imported `NuGet.targets` and on .NET Core we didn't have the file locally to copy it. Now that the import is explicit this file isn't needed.

Reply:
And to the location, it should be either in the VS install location (when `IsRunningFromVisualStudio` is true) or next to MSBuild in the .NET Core case (`$(MSBuildToolsPath)`). It should be the case that `MSBuildToolsPath` and `MSBuildExtensionsPath` are the same, but not necessarily. This work ok with mono?

Reply:
We deploy to `MSBuildExtensionsPath`. And that differs from `MSBuildToolsPath`. If the two are the same for netcore, then we can change it to use `MSBuildExtensionsPath`, I guess?

Reply:
Is it possible to change? Prefer to not use `MSBuildExtensionsPath`. Since they're fully required now, it's not really an extension. More like Roslyn is:

https://github.com/Microsoft/msbuild/blob/master/src/MSBuild/app.config#L101

The checked in file was just to get tests working not for any deployment strategy.

Reply:
re:change, sure, I'll do that. I am wondering if I should be removing it from `MSBuildExtensionsPath` or have it in both locations (it doesn't import anything, currently!).